src/main/res/values/strings.xml

```diff
@@ -96,12 +96,12 @@
     the applications it provides.</string>
   <string name="number_of_applications">Number of applications</string>
   <string name="ok">OK</string>
-  <string name="open_link_confirm_format">Open link %s?</string>
+  <string name="open_link_confirm_format">Open the %s link?</string>
```

Comment:
I think it's not good to allow to place the link between words. It may be really long. And still I'm against quotes.
"Open the link %s?" or "Open the following link: %s?", how about these options?